.kokoro/client/main.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	pb "github.com/jadekler/dummy-benchmarking-repo/.kokoro/grpc"
+	"google.golang.org/grpc"
+)
+
+const (
+	address = "localhost:50051"
+)
+
+func main() {
+	conn, err := grpc.Dial(address, grpc.WithInsecure())
+	if err != nil {
+		log.Fatalf("did not connect: %v", err)
+		fmt.Printf("did not connect: %v", err)
```

Comment:
remove, because:

1. The log.Fatalf will immediately cause the program to exit, so this line will never run
2. The log.Fatalf logs, so this would just duplicate the log (unnecessary)